### PR TITLE
fixed crappy arsenal.com/fixtures formatting issues again

### PR DESCRIPTION
**Pull request by ingenium21**
also added new case for !next whenever the date format is in %d/%m/%Y format instead of the usual %b %d format.

image

### CODE REVIEW

**Comment by ingenium21** [comment]
I still need to troubleshoot the matches happening in Florida, but at least this gets wengy working again.

File: cogs/fixtures.py [hunk 1]
```diff
@@ -205,10 +243,21 @@ def findFixtures(matches, number):
...
             comp = matches[i].find("div",{"class","event-info__extra"}).text
         except:
-            time = "TBD"
-            date = matches[i].find("div",class_=False, id=False).text.split(' ')
-            date = (date[0][:3] + " " + date[1]).split(',')[0]
-            comp = matches[i].find("div",{"class","event-info__extra"}).text
+            try:
+                date = matches[i].find("div",{"class","event-info__date"}).text
+                #date: '\n08/07/2022 Kick Off at 4.30pm BST\n'
+                date = date.split('\n')[1].strip()
```
Comment (FelixRandle) [inline cogs/fixtures.py:249]: You can just use .strip(), you don't need the splitting here

**Review by FelixRandle — APPROVED** [review]
Generally just adding more comments that explain what the hell is happening would help e.g.

```
date = matches[0].find("time").text
time = date.split('-')[1].strip()
date = date.split('-')[0][3:].strip()
```

Putting what format we expect date in here makes it a lot more readable

Also explaining what errors we're trying to catch with a try except helps, especially when there's like 15 lines that it could be coming from

Code wise looks fine though so I'll approve and let someone else choose if it should get merged

**Comment by ingenium21** [comment]
going to keep troubleshooting.

**Pull request closed without merging**